responsive-grid-layout/tugba/styles.css

```diff
@@ -0,0 +1,55 @@
+*{
+   margin: 0;
```

Comment:
I'd recommend to never set padding and margin to 0 for all elements (selector *)

Reply:
I wıll definitely remember this note.